roles/edpm_multipathd/tasks/install.yml

```diff
@@ -30,21 +30,31 @@
     - name: Gather services facts
       ansible.builtin.service_facts:
 
-    - name: Stop multipathd service on the host
-      ansible.builtin.systemd:
-        name: "{{ item }}"
-        state: stopped
-        enabled: false
+    - name: Stop multipathd on the host and restart edpm_multpathd if running
       when:
         - ansible_facts.services["multipathd.service"] is defined
         - ansible_facts.services["multipathd.service"]["status"] != "not-found"
         - ansible_facts.services["multipathd.service"]["status"] == "enabled"
-      failed_when: false
-      loop:
-        - multipathd.service
-        - multipathd.socket
-      loop_control:
-        index_var: multipath_service_index
+      block:
+        - name: Stop multipathd service on the host
+          ansible.builtin.systemd:
+            name: "{{ item }}"
+            state: stopped
+            enabled: false
+          loop:
+            - multipathd.service
+            - multipathd.socket
+          loop_control:
+            index_var: multipath_service_index
+        - name: Restart edpm_multipathd.service if running
```

Comment:
Please add a comment above L49 to explain why the install tasks may need to restart the containerized service. I would say it's required to clean up a problem where both the host and containerized version of the service are running at the same time.